index.html

```diff
@@ -25,15 +25,14 @@ <h2 class="project-tagline">ETL programming in Python</h2>
         <section class="main-content">
             <!-- Introduction Section -->
             <h1><a id="pygrametl---etl-programming-in-python" class="anchor" href="#pygrametl---etl-programming-in-python" aria-hidden="true"><span class="octicon octicon-link"></span></a>pygrametl - ETL programming in Python</h1>
-            <p>pygrametl (pronounced py-gram-e-t-l) is a Python framework that provides commonly used functionality for the development of Extract-Transform-Load (ETL) processes. It is open-source and released under a 2-clause BSD license.</p>
+            <p>pygrametl (pronounced py-gram-e-t-l) is a Python framework that provides functionality commonly used when developing Extract-Transform-Load (ETL) programs. It is fully open-source and released under a 2-clause BSD license. As shown in the figure below, an ETL program developed using pygrametl consists of SQL statements that create the data warehouse's schema (i.e., dimension tables, fact tables, and indices) and standard Python code that implements the ETL program itself.</p>
```

Comment:
Does the ETL program consist of SQL statements to create the DW schema?

Reply:
Reworded to state that `pygrametl` assumes the necessary tables have already been created.